src/components/apps/launch/messages.js

```diff
@@ -32,6 +32,7 @@ export default {
         advancedSettings: "Advanced Settings (optional)",
         analysisInfo: "Analysis Info",
         analysisName: "Analysis Name",
+        analysisNameHelp: "Enter analysis name",
```

Comment:
I think this could use more elaboration. Maybe let the user know that this analysis name will also become the first part of the output folder name (by default), and they can change it here.

Reply:
Not sure how to convey that concisely here. Might want to get that sorted out. I will defer that to Ux meeting.